cli/cmd/pull/options.go

```diff
@@ -7,9 +7,11 @@ var opts = &options{}
 
 type options struct {
 	AgentDigest string
+	OutputFormat      string
 }
 
 func init() {
 	flags := Command.Flags()
 	flags.StringVar(&opts.AgentDigest, "digest", "", "Digest of the agent to pull")
+	Command.Flags().StringVarP(&opts.OutputFormat, "output", "o", "json", "Output format (json|yaml)")
```

Comment:
lets not allow yaml outputs because we have to support then the marshallign from yaml on input (will break).
lets just use `--json` binary flag that will only print out json result when used `dirctl pull --digest <digest> --json`